spark/src/test/scala/org/apache/spark/sql/delta/DeltaTestUtils.scala

```diff
@@ -358,3 +363,48 @@ trait DeltaTestUtilsForTempViews
     }
   }
 }
+
+/**
+ * Trait collecting helper methods for DML tests e.p. creating a test table for each test and
+ * cleaning it up after each test.
+ */
+trait DeltaDMLTestUtils
+  extends DeltaTestUtilsBase  with BeforeAndAfterEach {
+  self: SharedSparkSession =>
+
+  protected var tempDir: File = _
+
+  protected var deltaLog: DeltaLog = _
+
+  protected def tempPath: String = tempDir.getCanonicalPath
+
+  override def beforeEach(): Unit = {
```

Comment:
While you're changing this, please see if you can set the access modifier to `protected` for afterEach and beforeEach. I was always annoyed by it not having the correct access modifier

Reply:
Done :+1: